uhubctl.c

```diff
@@ -388,6 +392,7 @@ static int get_hub_info(struct libusb_device *dev, struct hub_info *info)
             int bus = libusb_get_bus_number(dev);
             snprintf(info->location, sizeof(info->location), "%d", bus);
             int pcount = get_port_numbers(dev, port_numbers, MAX_HUB_CHAIN);
+            info->level = pcount + 1;
```

Comment:
From this code, `pcount` could be only `0` or more, it means level must be `1` or more.
In other words, `0` should be impossible value and is safe to use as wildcard.